distributed/distributed-schema.yaml

```diff
@@ -224,6 +224,11 @@ properties:
                   These modules will have a ``routes`` keyword that gets added to the main HTTP Server.
                   This is also a list that can be extended with user defined modules.
 
+          whitelist:
```

Comment:
Can we call this `allowlist` or `allow_list` (here and elsewhere)?

Reply:
That seems totally reasonable, I updated to use `allowed-imports`